cli/mfg.c

```diff
@@ -786,6 +788,191 @@ static int security_config_set(int argc, char **argv)
 	return 0;
 }
 
+#if HAVE_LIBCRYPTO
+static int load_sig_from_file(FILE *sig_fimg, uint8_t *sig)
```

Comment:
Same problems as with the other load functions. Also, why is this one special in that it's only in the CLI and the others get to be in the library?